app/models/dataset.rb

```diff
@@ -37,6 +37,10 @@ class Dataset < ApplicationRecord
   scope :owned_by, ->(creator_id) { where(creator_id: creator_id) }
   scope :published, ->{ where(status: "published") }
 
+  def self.columns
+    super.reject { |c| c.name == "stage" }
+  end
```

Comment:
Patch needed to avoid `PGError: ERROR: column "stage" does not exist`

ActiveRecord caches table columns, and uses this cache to build INSERT statements. So even if the code is not touching that column, ActiveRecord will still attempt to set it to `NULL` when saving models.

https://blog.codeship.com/rails-migrations-zero-downtime/